src/main/java/com/labsynch/labseer/service/StandardizationServiceImpl.java

```diff
@@ -777,7 +694,101 @@ public int restandardizeParentStructures(List<Long> parentIds)
 			// Update the percentage.
 			previousPercent = percent;
 		}
-		return parentIds.size();
+		return(p);
+	}
+	
+	@Transactional(propagation = Propagation.REQUIRES_NEW)
```

Comment:
Ok, so this one we are keeping transactions at the batch level.

Reply:
Yeah, this stuff isn't doing registrations and then searches. It's goal is to update quickly as all the chekcing has already been done. The bulk loader is different since it registeres "on the fly".